src/modules/extendedMap/i18n/fr_FR.json

```diff
@@ -29,20 +29,20 @@
                 "title": "Bâtiments rattachés"
             },
             "classrooms": {
-                "subtitle": "{total} in total, thereof {unavailable} under construction and {free} free",
+                "subtitle": "{total} au total, dont {unavailable} en construction et {free} disponible",
                 "table": {
-                    "countdown": "time left",
-                    "free": "Free Places",
-                    "freeClassrooms": "1 free classroom|{n} free classrooms",
+                    "countdown": "temps restant",
+                    "free": "Place libre|Places libre",
```

Comment:
this one needs to be `Places libre` only.
It is the heading of the table and will not be adjusted.

Reply:
it is to fix